.github/workflows/test.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
                   php-version: ${{ matrix.php-version }}
                   coverage: pcov
                   tools: composer:v2, pecl
-                  extensions: mongodb-1.9.0, soap
+                  extensions: mongodb-1.9.0, soap, gd2
```

Comment:
Dont forget you'll need a PR on dev-kit when it'll work.

Reply:
I just working on it.

Reply:
Can you look at https://github.com/sonata-project/dev-kit/pull/1450